src/Conversion/ONNXToKrnl/Tensor/Transpose.cpp

```diff
@@ -20,83 +20,192 @@ using namespace mlir;
 namespace onnx_mlir {
 
 struct ONNXTransposeOpLowering : public ConversionPattern {
+  using MDBuilder = MultiDialectBuilder<KrnlBuilder, IndexExprBuilderForKrnl,
```

Comment:
If you wanted to shorten `MultiDialectBuilder` to `MDBuilder` globally, I think that would improve readability as it kind of obvious that it is for multiple ones as we see the list in the template brackets. I understand that you are using it here in a `using`, which is another way to shorten it too.

Reply:
I leave this as it is at this moment since different levels of lowering need different builders.